exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/metrics_encoder/__init__.py

```diff
@@ -65,8 +65,17 @@ class OTLPMetricExporterMixin:
     def _common_configuration(
         self,
         preferred_temporality: Dict[type, AggregationTemporality] = None,
+        preferred_aggregation: Dict[type, "opentelemetry.sdk.metrics.view.Aggregation"] = None,
     ) -> None:
 
+        MetricExporter.__init__(
+            self,
+            preferred_temporality=self.get_temporality(preferred_temporality),
+            preferred_aggregation=self.get_aggregation(preferred_aggregation),
+        )
+
+    def get_temporality(self, preferred_temporality: Dict[type, AggregationTemporality]) -> Dict[type, AggregationTemporality]:
+
         instrument_class_temporality = {}
```

Comment:
Looks like this is guaranteed to be overwritten. Feel free to remove this line if you want (I don't mind it from a style perspective but linters might not like it). Ditto [line 135](https://github.com/open-telemetry/opentelemetry-python/pull/3603/files#diff-69723411a82d22ec56b486c8c744df22b6a4f245d68f057e17263d04d17b3dbeR135) below.

Reply:
Good catch! Removed on 49808de

Reply:
Thanks -- looks like the one on line 135 got removed but this one's still there?

Reply:
My mistake! removed on 5c065913e07ba3bd2fc79cf6bdeec65cf4aba9f7